### DISCUSSION TITLE
Mongo db entity, fields meaning

### DISCUSSION THREAD

**Discussion by mfilone** [phax/peppol-reporting#4] [Q&A] [answered]
We would like to use this library, for EUSR and TSR, and we are trying to understand what mongo entity fields mean.Can you describe them for us? for now we think they have the following meaning: 

- "direction": -> Incoming (document received from peppol), outgoing (document sent to peppol)
-  "c2Id": -> Peppol Identifier of document sender  (Service Provider value)
- "c3Id": -> Peppol Identifier of document receiver (Service Provider value)
- "c1CountryCode": -> Sender country code
- "c4CountryCode": -> Receiver  country Code       
- "endUserID": we don't have any idea....`

**Comment by phax** [comment, marked as answer]
These fields are take from the official specifications (https://docs.peppol.eu/edelivery/specs/reporting) as well as the additonal guideline document (https://docs.peppol.eu/edelivery/changelog/2023-08/2023-08-20%20Peppol%20Reporting%20-%20SP%20Guideline%20v2.0.pdf)

The fields are quickly explained in the main README at https://github.com/phax/peppol-reporting#data-collection

hth :)

**Reply by mfilone** [reply]
Thanks a lot

**Reply by mfilone** [reply]
As suggestion, for someone that not have pretty clear peppol fields meaning the readme description can't be totally clear, maybe you can add pdf ref in README.

**Reply by phax** [reply]
The PDF references are already there, at the top of the page - https://github.com/phax/peppol-reporting#peppol-reporting - maybe it's simply too much information
